### PR TITLE
Update rules_apple

### PR DESCRIPTION
**Pull request by keith**
This should fix https://github.com/bazelbuild/bazel/issues/10026

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -28,11 +31,3 @@ load(
 )
 
 apple_support_dependencies()
-
```
Comment (DavidGoldman) [inline WORKSPACE]: Why is this no longer needed?

Comment (keith) [reply WORKSPACE]: This is included by rules_apple by default now

**Comment by DavidGoldman** [comment]
Looks like you'll need to run the [update goldens script](https://github.com/bazelbuild/tulsi/blob/master/src/TulsiGeneratorIntegrationTests/update_goldens.sh)

**Comment by keith** [comment]
I actually can't get any of these tests to run before or after this change locally, maybe because I'm forcing a newer version of Xcode since I don't have 10.2 anymore?

```
> Critical error logged: Failed to query information about the Bazel workspace. Bazel exited with code 1
Details:
2019/10/14 20:24:29 could not get the user's cache directory: $HOME is not defined

> S: ** Completed get_package_path in 0.0231s
> Critical error logged: Was not able to extract the execution root from the workspace. This is a Tulsi or Bazel bug, please report.
> Critical error logged: Was not able to extract the bazel bin symlink name from the workspace. This is a Tulsi or Bazel bug, please report.
> I: Build Events JSON file at "/var/folders/q5/7_wh20z93c74nj3tllwjl5nr0000gn/T/build_events_23415.json"
> S: Extracting info for 2 rules
> I: Running /usr/local/bin/bazel --bazelrc=/dev/null build --announce_rc --show_result=0 --noshow_loading_progress --noshow_progress --symlink_prefix=/ --define=TULSI_TEST=dbg --host_force_python=PY2 --workspace_status_command=/usr/bin/true --ios_minimum_os=8.0 --macos_minimum_os=10.10 --tvos_minimum_os=10.0 --watchos_minimum_os=3.0 --xcode_version=10.2.1 --define=RULES_SWIFT_BUILD_DUMMY_WORKER=1 --strategy=SwiftCompile=local --apple_platform_type=ios --cpu=ios_x86_64 --watchos_cpus=i386 --announce_rc --override_repository=tulsi=/var/folders/q5/7_wh20z93c74nj3tllwjl5nr0000gn/T/test_tmp_dir.T2ifhg/EndToEndGenerationTests.xctest/Contents/Resources --compilation_mode=dbg --define=apple.add_debugger_entitlement=1 --define=apple.propagate_embedded_extra_outputs=1 --define=apple.experimental.tree_artifact_outputs=1 --features=debug_prefix_map_pwd_is_dot --tool_tag=tulsi:generator --build_event_json_file=/var/folders/q5/7_wh20z93c74nj3tllwjl5nr0000gn/T/build_events_23415.json --noexperimental_build_event_json_file_path_conversion --noexpand_test_suites --aspects @tulsi//:tulsi/tulsi_aspects.bzl%tulsi_sources_aspect --output_groups=tulsi_info,-_,-default //TestSuite:TestApplication //TestSuite:local_tagged_tests
> S: ** Completed bazel_extract_source_info in 0.0242s
> Critical error logged: Failed to retrieve information about the Bazel workspace. This usually means that the contents of a BUILD file are incorrect or flags are missing. Read the message log carefully for details.
Details:
```

**Comment by DavidGoldman** [comment]
It's alright I will try to update the goldens myself

**Comment by keith** [comment]
Ok it looks like it did work once I downloaded 10.2.1, although that error being fixed definitely surprises me. I've pushed the update, hopefully it passes CI

**Comment by keith** [comment]
All green now!

**Comment by meisterT** [comment]
I verified that it fixes the issue, see https://github.com/bazelbuild/bazel/issues/10026#issuecomment-542184985

**Pull request closed without merging**